eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -116,7 +116,7 @@ jobs:
         CoverageArg: $(CoverageArg)
         PythonVersion: $(PythonVersion)
         BuildTargetingString: ${{ parameters.BuildTargetingString }}
-        ToxTestEnv: 'whl,sdist'
+        ToxTestEnv: 'whl,sdist,latestdependency,minimumdependency'
```

Comment:
Leave this for testing, I don't want to introduce additional build time _yet_.

Reply:
I will remove this additional test. Added it to verify on pipelines. Re -running entire repo test on my machine now. I will remove these tests and push once test is completed.